src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ControllerExtensions.cs

```diff
@@ -60,7 +60,7 @@ public static class ControllerExtensions
             }
 
             var allOtherShells = allSettings.Where(t => !String.Equals(t.Name, model.Name, StringComparison.OrdinalIgnoreCase));
-            if (allOtherShells.Any(t => String.Equals(t.RequestUrlPrefix, model.RequestUrlPrefix?.Trim(), StringComparison.OrdinalIgnoreCase) && (t.RequestUrlHost?.Contains(model.RequestUrlHost, StringComparison.OrdinalIgnoreCase) ?? false)))
+            if (allOtherShells.Any(t => (model.RequestUrlPrefix?.Trim()?.Equals(t.RequestUrlPrefix, StringComparison.OrdinalIgnoreCase) ?? false) && (t.RequestUrlHost?.Contains(model.RequestUrlHost, StringComparison.OrdinalIgnoreCase) ?? false)))
```

Comment:
I don't know why I got NRE when `RequestUrlPrefix` is `null` I just tweek it a little bit to make it work